api/app/src/command/medical/document/document-query.ts

```diff
@@ -110,18 +112,25 @@ export const updateDocQuery = async ({
     });
 
     if (existing) {
```

Comment:
nit: I know this was here before, but let's try to use descriptive variable names. When looking at the PR, I thought "what's existing?" and had to expand the code and look at the logic.

Using descriptive variable names, like `existingPatient` for example, saves time & cognitive overhead.